compiler/ballerina-lang/src/main/resources/ballerina-toml-schema.json

```diff
@@ -121,6 +121,12 @@
             "type": "string"
           }
         },
+        "includes": {
```

Comment:
Shall we make this `include` instead of `includes`? This is to be consistent with the imperative mood we have used for `export`